dev/drivers/scripts/plots/mesoscale/jevs_mesoscale_grid2obs_plots.sh

```diff
@@ -36,7 +36,7 @@ export VERIF_CASE="grid2obs"
 export MODELNAME=${COMPONENT}
 
 # EVS Settings
-export HOMEevs="/lfs/h2/emc/vpppg/noscrub/${USER}/EVS"
+export HOMEevs="/lfs/h2/emc/vpppg/noscrub/${USER}/EVS_mesoscale_fixes_v2/EVS"
```

Comment:
Revert to `export HOMEevs="/lfs/h2/emc/vpppg/noscrub/${USER}/EVS"`

Reply:
Fixed.